cvat/apps/iam/templates/account/email/email_confirmation_message.html

```diff
@@ -121,15 +132,12 @@
         <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600px;">
           <tr>
             <td align="center" valign="bottom">
-              <a href="https://www.cvat.ai/" target="_blank" style="display: inline-block;">
-                <img
-                  src="https://github.com/cvat-ai/cvat/blob/develop/site/content/en/images/cvat_poster_with_name.png?raw=true"
-                  alt="Logo"
-                  border="0"
-                  width="100%"
-                  style="display: block; width: 100%;"
-                >
-              </a>
+              <svg style="margin: 24px;" width="128" height="24" viewBox="0 0 80 15" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Have you checked if email clients can actually display this? There's some evidence that inline SVG support in various email clients is lacking: https://css-tricks.com/a-guide-on-svg-support-in-email/#svg-email-inline https://www.caniemail.com/features/html-svg/

Reply:
On the other side, it seems, src="base64" is also not well-supported..

Reply:
Works for me.